examples/mini-app/src/components/game/InfoSheetPanel.tsx

```diff
@@ -28,7 +28,9 @@ export function InfoSheetPanel({
         <div className="grid grid-cols-2 gap-x-4 sm:gap-x-6 gap-y-3 sm:gap-y-4 text-sm font-medium text-text-on-surface-variant">
           <div>
             <p>Win chance:</p>
-            <p className="text-text-on-surface">{winChance}%</p>
+            <p className="text-text-on-surface">
+              {winChance !== undefined ? `${winChance}%` : "See game screen"}
```

Comment:
Can winChance be calculated dynamically based on selections? I.e. can our component see game screen and show a few comma-separated numbers or a sum of chances for each output that we consider a win?